docs/blog/2023-08-15-keep-your-secrets.md

```diff
@@ -0,0 +1,120 @@
+Here’s why it stands out:
+
+* **Ease of Use**: With a [**Quickstart Guide**](https://vsecm.com/docs/quickstart/), 
+  intuitive [**CLI**](https://vsecm.com/docs/cli/?ref=zerotohero.dev), and 
+  developer-friendly [**SDK**](https://vsecm.com/docs/sdk/), **VSecM** 
+  simplifies secrets management without compromising security.
+* **Kubernetes Is a First-Class Citizen**: **VSecM** seamlessly integrates with 
+  Kubernetes, leveraging [**SPIRE**](https://spiffe.io/docs/latest/spire-about/) 
+  for authentication and offering flexible secret storage and transformation options.
+* **Community Engagement**: We believe in open collaboration and community-driven 
+  development. Our [**Contributing Guide**](https://vsecm.com/docs/contributing/) 
+  invites developers to get involved, ask questions, and contribute to the project.
+* **Secure by Default**: **VSecM** offers advanced security features, including 
+  encryption, manual master secret setting, and integration with various backing 
+  stores.
+
+## Join the Future of Secrets Management
+
+**VMware Secrets Manager** is more than just a tool; it’s a community-driven project 
+that aims to redefine secrets management in a cloud-native world. We invite you to 
+explore **VSecM**, contribute to its growth, and join us in shaping the future of 
+secure and resilient applications.
+
+Whether you’re a developer looking to contribute or an organization seeking a 
+flexible and secure solution for secrets management, **VSecM** offers a welcoming 
+and innovative platform.
+
+### This is Just the Beginning: A Vision for the Future
+
+At **VMware Secrets Manager**, we believe in continuous innovation and growth. 
+Our [**roadmap**](https://vsecm.com/docs/roadmap) is a testament to our commitment 
+to delivering cutting-edge solutions that meet the evolving needs of the cloud-native 
+community.
+
+Here's a glimpse of what's on the horizon:
+
+* **Stability and Usability Enhancements**: Upcoming releases focus on improving 
+  stability, documentation, build automation, and overall user experience.
+* **Innovative Features**: From Secretless VSecM to machine-readable logs, 
+  performance improvements, and large-scale secret ingestion, we're pushing the 
+  boundaries of what's possible in secrets management.
+* **Integration and Compatibility**: Future iterations include integration with 
+  AWS KMS, Azure Key Vault, Google Cloud KMS, and even HashiCorp Vault, expanding 
+  the reach and compatibility of VSecM.
+* **Community Development and Collaboration**: We're investing in community 
+  development efforts, multi-cluster secret federation, and creating abstractions 
+  to make cloud integrations easier.
+
+Our roadmap is not just a plan; it’s a promise to our users and contributors that 
+we have a lot planned for the future of **VMware Secrets Manager**. 
+
+We’re excited about the journey ahead and invite you to be a part of it.
+
+> **⭐️ Star Us on GitHub ⭐️**
```

Comment:
nitpick: is it intentionally twice?

Reply:
:) was intentional.

Actually, I was unsure to put it left, or right. But the more stars the better :D .

Reply:
Ah yes, it’s intentional. 

I want it to be visible above the fold, and then near the and after scrolling.